### PR TITLE
[KO] Updated Concurrency section

### PR DESCRIPTION
**Pull request by devleoper**
Related: #667 and #201

Any ideas or reviews?
@Dalgona, @malkoG, @marocchino, @riseshia

### CODE REVIEW

File: ko/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -64,6 +66,8 @@ iex> send pid, :ok
 :ok
 ```
 
+`listen/0` 함수를 재귀적으로 만들어 여러 메시지를 처리할 수 있습니다. 스스로 호출하지 않으면 프로세스는 첫 메시지를 처리한 후에 종료될 것입니다.
```
Comment (riseshia) [inline ko/lessons/advanced/concurrency.md]: `listen/0` function is recursive
so, it's better to say "`listen/0` 함수를 재귀적으로 호출하므로" I think

**Comment by riseshia** [comment]
i'm done XD

**Comment by marocchino** [comment]
LGTM

**Comment by malkoG** [comment]
Nice :)

**Pull request merged**